### DISCUSSION TITLE
Query Error: request to https://server-name failed, reason: self-signed certificate in certificate chain

### DISCUSSION THREAD

**Discussion by ATHENA12D** [Budibase/budibase#12775] [Help]
Hi Guys,
Recently I developed Budibase by Docker Run method, so far everything is perfect. I created an application and created a query to make GET request using Rest API on this URL https://server-name , but I got this error more than once "Query Error: request to https://server-name failed, reason: self-signed certificate in certificate chain".
My question is: How can I put the self certificate in this URL https://server-name so that the Budibase can find it when i make inquiring about this domain?

**Comment by ConorWebb96** [comment]
Closing as this is a duplicate of https://github.com/Budibase/budibase/discussions/12777
